pages/linux/paccache.md

```diff
@@ -0,0 +1,19 @@
+# paccache
+
+> A flexible pacman cache cleaning utility.
+
+- Remove all but the 3 most recent package versions from the pacman cache:
+
+`paccache -r`
+
+- Set the number of package versions to keep:
+
+`paccache -rk {{num_versions}}`
+
+- Perform dryrun and show the number of candidate packages:
+
+`paccache -d`
+
+- Move candidate packages to a directory instead of deleting them:
+
+`paccache -m {{directory}}`
```

Comment:
Could `directory` be `path/to/directory` please? It'll make it more consistent with other pages.

Reply:
Sounds good too, I will change it.